File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/UserFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.User;

public class UserFilterValidator extends FilterValidator<User> {

    public UserFilterValidator(CustomValidator<User> validator) {
        super(User.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/UserFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.User
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class UserFilterValidator(validator: Validator<User>) : FilterValidator<User>(
    User::class, validator)
